site/server/app/com/fortysevendeg/exercises/persistence/repositories/UserProgressRepository.scala

```diff
@@ -53,14 +64,14 @@ class UserProgressDoobieRepository(implicit persistence: PersistenceModule) exte
   override def findByUserId(userId: Long): ConnectionIO[List[UserProgress]] =
     persistence.fetchList[Long, UserProgress](Q.findByUserId, userId)
 
-  override def findByExerciseVersion(
+  override def findByUserIdAggregated(userId: Long): ConnectionIO[List[(String, Long, Boolean)]] =
+    persistence.fetchList[Long, (String, Long, Boolean)](Q.findByUserIdAggregated, userId)
```

Comment:
We should create type aliases perhaps for these pairs of input and tupled replacements if these are in use elsewhere.